src/OrchardCore.Modules/OrchardCore.Contents/Controllers/ApiController.cs

```diff
@@ -33,7 +35,74 @@ public async Task<IActionResult> Get(string contentItemId)
                 return Unauthorized();
             }
 
-            return new ObjectResult(contentItem);
+            return Ok(contentItem);
+        }
+
+        [HttpDelete]
+        [IgnoreAntiforgeryToken]
```

Comment:
Out of curiosity, how is authentication managed for these API controllers?

If you simply rely on the default authentication scheme (cookies, configured by the Users module), then disabling antiforgery validation is terrible, as it will allow anyone to forge malicious requests (same-site would help block them, but it's not supported everywhere and we have to disable it completely in the OpenID module).